docs/pages/modules/get-started.mdx

```diff
@@ -0,0 +1,128 @@
+The presence of the module config is enough for [Expo Autolinking](https://docs.expo.dev/workflow/glossary-of-terms/#expo-autolinking) to recognize it as an Expo module and automatically link your native code.
+
+### Add the `expo-modules-core` native dependency
+
+Add `expo-modules-core` as a dependency in your podspec and **build.gradle** files.<br/>
+
+<CodeBlocksTable tabs={["*.podspec", "build.gradle"]}>
+
+```ruby
+# ...
+Pod::Spec.new do |s|
+  # ...
+  s.dependency 'ExpoModulesCore'
+end
+```
+
+```groovy
+// ...
+dependencies {
+  // ...
+  implementation project(':expo-modules-core')
+}
+```
+
+</CodeBlocksTable>
+
+### Add Expo packages to dependencies
+
+Add `expo` package as a peer dependency in your **package.json** — we recommend using `*` as a version range so as not to cause any duplicated packages in user's **node_modules** folder. Your library also needs to depend on `expo-modules-core` but only as a dev dependency — it's already provided in the projects depending on your library by the `expo` package with the version of core that is compatible with the specific SDK used in the project.<br/>
+
+<CodeBlocksTable tabs={["package.json"]}>
+
+```json
+{
+  // ...
+  "devDependencies": {
+    "expo-modules-core": "^X.Y.Z"
+  },
+  "peerDependencies": {
+    "expo": "*"
+  }
+}
+```
+
+</CodeBlocksTable>
+
+## Create a native module
+
+To use the Expo Modules API for native modules, you need to [set up your library as an Expo module](#set-up-a-library-as-an-expo-module). Once complete, create Swift and Kotlin files from the templates below.
+
+<CodeBlocksTable tabs={["MyModule.swift", "MyModule.kt"]}>
+
+```swift
+import ExpoModulesCore
+
+public class MyModule: Module {
+  public func definition() -> ModuleDefinition {
+    // Definition components go here
+  }
+}
+```
+
+```kotlin
+package my.module.package
+
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+
+class MyModule : Module() {
+  override fun definition() = ModuleDefinition {
+    // Definition components go here
+  }
+}
+```
+
+</CodeBlocksTable>
+
+Then, add your classes to iOS and/or Android `modules` in the [module config](./module-config.mdx). Expo Autolinking will automatically link these classes as native modules in the user's project.
+
+<CodeBlocksTable tabs={["expo-module.config.json"]}>
+
+```json
+{
+  "ios": {
+    "modules": ["MyModule"]
+  },
+  "android": {
+    "modules": ["my.module.package.MyModule"]
+  }
+}
+```
+
+</CodeBlocksTable>
+
+If you already have an example app in your workspace, ensure that the module is linked correctly.
+- **On iOS** you need to run `pod install` to link the new class.
+- **On Android** the native module class will be linked automatically before building, as part of the Gradle build task.
```

Comment:
```suggestion
- **On Android** the native module class will be linked automatically before building, as part of the Gradle build task.
- **On iOS** you need to run `pod install` to link the new class.
```

We should preferably list Android before iOS, see:
*https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#referencing-android-ios-and-web